### ISSUE TITLE
ENH: contributors update Change log with merged PR

### ISSUE THREAD

**Issue by nathanshammah** [qutip/qutip#1055]
Once a Pull Request is merged to QuTiP, the contributor could add a short description to the Changelog, which contains changes of each [version release](http://qutip.org/docs/latest/changelog.html). 
It could be in the style of current changelog:

* A one-liner for **minor changes**, with a descriptive text and the author's name. 
* A short paragraph for **major changes**. 

This guideline could be added to the contribution [how-to](https://github.com/qutip/qutip-doc/blob/master/qutip_dev_contrib.md ).

**Comment by hodgestar** [comment]
Superseded by #1894.
